web/satellite/src/components/common/VButton.vue

```diff
@@ -70,6 +76,10 @@ export default class VButton extends Vue {
 
         if (this.isBlueWhite) return 'container blue-white';
 
+        if (this.isWhiteGreen) return 'container white-green';
+
+        if (this.isGreenWhite) return 'container green-white';
```

Comment:
It looks like this should use a enum for the coloring instead.

Reply:
Tests are also missing for this new behavior for VButton.

Sidenote, when converting from bools to enum, use a separate change request rather than including them here. Because otherwise a single pull-request gets too large for review.

Reply:
Hi Egon, The way this was implemented was based on the existing design pattern of VButton. Before moving forward with the change, I was wondering what the benefits are for changing this design to use an enum rather than the current pattern?